Application/Api/Education/Certificate/Generator/Repository/ESBD/EsbdStyle.php

```diff
@@ -0,0 +1,893 @@
+                    ))
+            )->styleMarginTop($MarginTop)
+            ->styleHeight($Height);
+
+        return $DescriptionSlice;
+    }
+
+    /**
+     * @param $personId
+     * @param bool $isMissing
+     *
+     * @return Slice
+     */
+    protected function getDescriptionHeadConsumer($personId, $isMissing = false, $MarginTop = '15px')
+    {
+        $DescriptionSlice = (new Slice());
+        if ($isMissing) {
+            $DescriptionSlice->addSection((new Section())
+                ->addElementColumn((new Element())
+                    ->setContent('Bemerkungen:')
+                    , '16%')
+                ->addElementColumn((new Element())
+                    ->setContent('Fehltage entschuldigt:')
+//                    ->styleBorderBottom('1px')
+                    ->styleAlignRight()
+                    , '25%')
+                ->addElementColumn((new Element())
+                    ->setContent('{% if(Content.P' . $personId . '.Input.Missing is not empty) %}
+                                    {{ Content.P' . $personId . '.Input.Missing }}
+                                {% else %}
+                                    &nbsp;
+                                {% endif %}')
+//                    ->styleBorderBottom('1px')
+                    ->styleAlignCenter()
+                    , '10%')
+                ->addElementColumn((new Element())
+                    ->setContent('unentschuldigt:')
+//                    ->styleBorderBottom('1px')
+                    ->styleAlignRight()
+                    , '25%')
+                ->addElementColumn((new Element())
+                    ->setContent('{% if(Content.P' . $personId . '.Input.Bad.Missing is not empty) %}
+                                    {{ Content.P' . $personId . '.Input.Bad.Missing }}
+                                {% else %}
+                                    &nbsp;
+                                {% endif %}')
+//                    ->styleBorderBottom('1px')
+                    ->styleAlignCenter()
+                    , '10%')
+                ->addElementColumn((new Element())
+                    ->setContent('&nbsp;')
+//                    ->styleBorderBottom('1px')
+                    ->styleAlignCenter()
+                    , '4%')
+            )
+                ->styleMarginTop($MarginTop);
+        } else {
+            $DescriptionSlice->addSection((new Section())
+                ->addElementColumn((new Element())
+                    ->setContent('Bemerkungen:'))
+            )->styleMarginTop($MarginTop);
+        }
+        return $DescriptionSlice;
+    }
+
+    /**
+     * @param $personId
+     * @param string $Height
+     * @param string $MarginTop
+     * @param string $PreRemark
+     * @param string|bool $TextSize
+     * @return Slice
+     */
+    protected function getDescriptionContentConsumer($personId, $Height = '150px', $MarginTop = '0px', $PreRemark = '', $TextSize = false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incorrect spacing between argument "$Height" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)